### ISSUE TITLE
RP2 crash with minimizing window in the designer

### ISSUE THREAD

**Issue by cpyrgas** [X-Sharp/XSharpPublic#756]
https://www.xsharp.eu/forum/german/2624-report-pro-2-laufzeitfehler-wenn-reportfenster-minimiert-wird#19266

Start Rp2Designer, i use 2.36.0.4, e.g. open "recent files" any rpt, click in the form the minimize button in the caption, and:
Problemsignatur:
Problemereignisname: CLR20r3
Problemsignatur 01: ReportPro2.exe
Problemsignatur 02: 2.36.0.4
Problemsignatur 03: 5f6865ab
Problemsignatur 04: ReportPro2.Runtime
Problemsignatur 05: 2.36.0.4
Problemsignatur 06: 5f6865a4
Problemsignatur 07: 34e
Problemsignatur 08: c8
Problemsignatur 09: System.AccessViolationException
Betriebsystemversion: 6.3.9600.2.0.0.256.103
Gebietsschema-ID: 1031
Zusatzinformation 1: 3931
Zusatzinformation 2: 39318975b0826d60fbcc811b5674ab1a
Zusatzinformation 3: c819
Zusatzinformation 4: c8193034f71ec652fa0054bf97b4896c

**Comment by cpyrgas** [comment]
Problem was the pointer dereferencing code in this line of RpReportDesigner.prg

SendMessage(oEvent:hWnd,WM_ICONERASEBKGND,DWORD(sPaint:hDC),0)

the correct code is 

SendMessage(oEvent:hWnd,WM_ICONERASEBKGND,DWORD(_CAST, sPaint:hDC),0)

and I checked the code in VO, where it is correct. So this problem was introduced probably when porting the RP2 code from VO to .Net. I searched for other similar errors in the code but could not find any.
